pacman_contest_cluster.py

```diff
@@ -236,6 +240,11 @@ def load_settings():
         settings['include_staff_team'] = False
         settings['staff_teams_dir'] = 'None'
 
+    if args.staff_teams_vs_others_only:
```

Comment:
Great, we need to do #49 and #55 as this is becoming a mess with so many options.. :-)